test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -688,13 +688,55 @@
 #      initPods: 20000
 #      measurePods: 5000
 
-- name: Unschedulable
+# Measure the time it takes to schedule default measurePods after scheduling 20% of high priority Pods that trigger preemption.
+# The measurePods are expected to be scheduled after scheduler handled the higher priority Pods. It's expected that running the apiserver updates asynchronously https://github.com/kubernetes/kubernetes/issues/126858 (once fixed), should significantly speed up this test.
+- name: PreemptionBlocking
   workloadTemplate:
   - opcode: createNodes
     countParam: $initNodes
   - opcode: createPods
     countParam: $initPods
-    podTemplatePath: config/templates/pod-large-cpu.yaml
+    podTemplatePath: config/templates/pod-low-priority.yaml
+  - opcode: createPods
+    countParam: $blockingPods
+    podTemplatePath: config/templates/pod-high-priority.yaml
+    skipWaitToCompletion: true
+  - opcode: createPods
+    countParam: $measurePods
+    collectMetrics: true
```

Comment:
I think this won't show the improvement in throughput. Scheduling throughput measurements start when at least one pod got scheduled. In this case, first measure pod will be scheduled **after** all high priority pods pass preemption process and are scheduled. You could think of creating the measurePods gradually, starting even before creating high priority pods.

Reply:
Yeah, I was also wondering how exactly the time will get measured, but I assumed it will start once the measured operation starts. I will take closer look, but I initially thought about starting the time measure (somehow) in the previous operation (scheduling preemption) and continue in the following. Obviously it would require code changes, but does it sound good? 

Not sure what you mean by creating measurePods gradually though.

Reply:
Wrapped both operations into time measurement.